server/src/main/java/io/pravega/schemaregistry/storage/impl/SchemaChunks.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.storage.impl;
+
+import io.pravega.common.util.ByteArraySegment;
+import io.pravega.schemaregistry.contract.data.SchemaInfo;
+import lombok.Data;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class SchemaChunks {
```

Comment:
Is this an older version of `ChunkUtils` above? I don't see this class used in the code here yet.

Reply:
removed.